custom_components/frigate/media_source.py

```diff
@@ -437,13 +437,43 @@ def from_str(
         if len(parts) < 2 or parts[1] != cls.get_identifier_type():
             return None
 
+        # NOTE: In Frigate 0.11.1 and previous versions, recordings were stored in
+        # a folder format /recordings/%Y-%m/%d/%h in local server time. In frigate
+        # 0.12+ the recordings are now stored in UTC time. In order to avoid a
+        # breaking change the folder format converted to /recordings/%Y-%m-%d/%h
+        # and both need to be handled for backwards compatibility.
+        # TODO probably a better way to do this using Frigate APIs with recordings info
```

Comment:
What does this TODO actually mean? If it's saying we could use the new recordings summary endpoint instead of directory navigation, we could but is the old endpoint going away?

I'd recommend either deleting the TODO, or make it more specific and put your name in it if you're signing up for it ;-)

Reply:
I'll make an issue instead